lms/envs/common.py

```diff
@@ -2120,6 +2117,13 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'openedx.core.djangoapps.site_configuration.middleware.SessionCookieDomainOverrideMiddleware',
 ]
 
+if django.VERSION >= (3, 1):
```

Comment:
Documenation says it should be top in middlewares.

Thats why appending seems wrong option